examples/plugins/using-plugins.rst

```diff
@@ -0,0 +1,37 @@
+Using MLflow Plugins
+--------------------
+
+MLflow plugins are simply Python packages that can be installed via PyPI or conda. In this example,
+we'll install a tracking plugin from source and use it within an example script.
+
+Install the Plugin
+~~~~~~~~~~~~~~~~~~
```

Comment:
Use a consistent tense for the verb in the heading. All your others are gerunds (end in "ing"). Therefore, this one should be "Installing the Plugin".

FWIW, we have done away with gerunds in the Databricks docs.